.github/workflows/check-sdk-compat.yml

```diff
@@ -0,0 +1,84 @@
+name: Check sdk-go compatibility
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_ref:
+        description: sdk-go ref to check ("latest" for latest release tag)
+        required: true
+        default: latest
+      api_ref:
+        description: api-go ref to check
+        required: true
+        default: master
+  workflow_call:
+    inputs:
+      sdk_ref:
+        description: sdk-go ref to check ("latest" for latest release tag)
+        required: true
+        default: latest
+      api_ref:
+        description: api-go ref to check
+        required: true
+        default: master
+
+jobs:
+  check-sdk-go-compatibility:
+    name: "Check sdk-go compatibility"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Validate inputs
```

Comment:
Probably redundant since the checkout step will fail if the ref isn't valid.

Reply:
I need to at least translate the "latest" input to an actual tag. The `checkout` step fails with the "latest" ref.

Reply:
Got it, yeah, I had it in my head that we want to verify master and not latest, but I think latest is right.
We need to give guidance though that sometimes `latest` is expected to fail, e.g. when making backwards incompatible changes, and that the action should be rerun with `master`. If that also fails, we should not cut a release.